template/default.nix

```diff
@@ -16,6 +16,10 @@
       stackable-secret-operator = attrs: {
         buildInputs = [ pkgs.protobuf pkgs.rustfmt ];
       };
+      stackable-opa-user-info-fetcher = attrs: {
+        # TODO: why is this not pulled in via libgssapi-sys?
+        buildInputs = [ pkgs.krb5 ];
+      };
```

Comment:
note(non-blocking): We should only put common stuff in the templated `default.nix`. All other operator-specific stuff should _not_ be templated and should live in the operator repositories. @NickLarsenNZ will look into that.

Reply:
Issue created here: https://github.com/stackabletech/operator-templating/issues/456